tsconfig.json

```diff
@@ -1,22 +1,21 @@
 {
 	"compilerOptions": {
 		"target": "es2020",
-		"baseUrl": ".",
```

Comment:
`baseUrl` is deprecated and has no effect in TypeScript>5. It just requires setting relative path specifiers in the rest of the tsconfig, as done below.